main.py

```diff
@@ -2,4 +2,4 @@
 x = 1
 y = x * 2
 z = [x for x in range(0, y-1)]
-print(x)
+print("number is: ", x)
```

Comment:
Hey Ava/Florian! I am not sure who made the changes here, but thanks anyway! It makes the print statement much better. Who likes, printed numbers without comments, right?

I think we should specify that it's `x` and not any other number since we have more numbers there. Just to avoid confusion.

Reply:
It was Ava! This line is getting cluttered. How about replacing the message with "x = " ?

Reply:
I agree, that's much better! LGTM